src/main/java/net/datafaker/transformations/JsonTransformer.java

```diff
@@ -53,31 +55,43 @@ public String generate(Iterable<IN> input, Schema<IN, ?> schema) {
             throw new IllegalArgumentException("The sequence should be finite of size");
         }
 
-        StringJoiner data = new StringJoiner(LINE_SEPARATOR);
+        StringBuilder sb = new StringBuilder();
```

Comment:
Why do we need to remove usage of line separator here?

I agree that there might be cases when it is preferred to get result without it however sometimes it is better to have it.

If you really need to have result without it then i would suggest to add dedicated method to json transformer builder to set lineseparator, then both old and new behavior will be supported